docs/operations/metrics.md

```diff
@@ -206,6 +206,7 @@ These metrics are for the Druid Coordinator and are reset each time the Coordina
 |`segment/dropQueue/count`|Number of segments to drop.|server.|Varies.|
 |`segment/size`|Size in bytes of available segments.|dataSource.|Varies.|
 |`segment/count`|Number of available segments.|dataSource.|< max|
+|`segment/totalCount`|Total number of available segments on each historical node.||< max|
```

Comment:
Is this a metric for historicals? Then it seems in a wrong section. These are the metrics for the coordinator.